proposals/nixpkgs-security.md

```diff
@@ -0,0 +1,278 @@
+
+- [ ] Preparatory fundamental upstream features (systemd and more): 4w, 10K EUR.
+- [ ] NixOS's default UEFI Secure Boot images via shim and shim-review: 4w, 10K EUR (*)
+- [ ] NixOS RFC "Bootspec v2": 2w, 4K EUR
+- [ ] Lanzaboote and TPM2: 4w, 10K EUR
+- [ ] Safe updates (A/B schemas) in NixOS with systemd-boot: 2w, 5K EUR
+- [ ] Safe Nix store via integrity techniques (*-verity): 4w, 10K EUR
+- [ ] Safe NixOS via maximal reduction of attack surface through open interpreters: 4w, 10K EUR.
+- Total: 59K EUR.
+
+(*) This is dependent on the [shim-review project](https://github.com/rhboot/shim-review) owners to accept our application.
+
+Explaination:
+
+In order to make many of those features a stable and reliable reality, some work has to be done upstream
+which will benefit the whole ecosystem and community and show the talent and expertise of our ecosystem.
+
+Then, a lot of work would be performed in parallel via lanzaboote maintainers
+but new developers who joined the project because they were seduced in the value proposition of combining NixOS and those features.
+
+Because this work is particularly low-level and difficult, those rates are a minimum for the quality standards we set in nixpkgs,
+it will probably entail many back'n'forth with UEFI ecosystem libraries in Rust as this is the tool of choice for
+the development of our tools.
+
+Finally, the last three points enable NixOS as, what we call, an "appliance operating system", that is, a highly-secure and locked-down
+deployment of NixOS in the field, e.g. Internet of Things, application servers, where upgrades can only be performed by replacing
+completely the whole system, in a safe way supporting safe rollbacks in case of failed upgrades but not supporting arbitrary
+downgrades to vulnerable versions. We would also aim to remove any potential "bugdoors", be it a Python interpreter which can run
+arbitrary Python code which are contained in NixOS for early boot due to flexibility and simplicity of the build story.
+
+
+- [ ] Web service scaffholding (architecture, structure, testing, packaging, deployment) and core features: APIs & ingesting (3w, 10K EUR)
+- [ ] [label-tracker](https://git.eno.space/label-tracker.git/) integration and improvements (2w, 5K EUR)
+- [ ] User story "Security team" work (2w, 5K EUR): filtering, triaging, generating advisories, linking metadata
+- [ ] User story "Maintainer" work (3w, 5K EUR): assisting security team, processing their own software, validating/participating, automation for PRs (bump, backport, mark EOL/vulnerable)
+- [ ] User story "End-user" work (2w, 5K EUR): receiving data about the current security story of a release, subscribing to security updates for a release, various APIs to reuse alerting, open data and exports
+
+- Total: 30K EUR
+
+Explaination:
+
+The working mode we envision is a strong web developer with extensive Nix experience developing a reusable high standard platform with
```

Comment:
Who is that web developer? One of the maintainers?

Reply:
It's whoever fits that description and is willing to put work into that